app/helpers/lead_partner_helper.rb

```diff
@@ -4,7 +4,22 @@ module LeadPartnerHelper
   def lead_partner_urn_and_location(lead_partner)
     [
       "URN #{lead_partner.urn}",
-      [lead_partner.school&.town, lead_partner.school&.postcode].select(&:present?).join(", "),
-    ]
+      [lead_partner.school&.town, lead_partner.school&.postcode].select(&:present?),
+    ].flatten.join(", ")
+  end
+
+  def lead_partner_row(not_applicable: false)
+    mappable_field(
+      not_applicable ? t(:not_applicable) : lead_partner_detail(lead_partner),
+      t("components.school_details.lead_partner_key"),
+      change_paths(:lead),
```

Comment:
`change_paths` with the `:lead` key is currently pointing to a lead school route.